docs/api/wrapper/setMethods.md

```diff
@@ -1,7 +1,11 @@
 ## setMethods
 
-::: warning
+::: warning Deprecation warning
 `setMethods` is deprecated and will be removed in future releases.
+
+There's no clear path to replace `setMethods`, because it really depends on your previous usage. It easily leads to flaky tests that rely on implementation details, which [is discouraged](<(https://github.com/vuejs/rfcs/blob/668866fa71d70322f6a7689e88554ab27d349f9c/active-rfcs/0000-vtu-api.md#setmethods)>).
```

Comment:
We can suggest users to use `jest.spyOn(Component.methods, 'method')`, which works most of the times. Ofc this will not work with the new composition api 😆 but if you are using that, you can figure it out.

Reply:
yep, assume composition API ppl are advanced users and can figure it out

Reply:
So you think that mentioning the ability to spy on methods is worth adding?

something along the lines of…

```
If you need to stub out a complex method, extract it out and test it.
On the other hand, if you want to assert a method is called, you can use your test runner to spy on it.
```

Reply:
I like that! but either way is fine, I think.

Reply:
done!